src/api.jl

```diff
@@ -210,7 +210,7 @@ greater than `atol`. To see details about the distance function, see [`podistanc
 * `atol` : minimal distance between two periodic orbits for them to be considered unique.
 
 """
-function uniquepos(pos::Vector{PeriodicOrbit}; atol::Real=1e-6)
+function uniquepos(pos::Vector{<:PeriodicOrbit}; atol::Real=1e-6)
```

Comment:
I guess you corrected this because you tried to call `uniquepos` and it errored, right? if so, we should have had a test that calls `uniquepos`. Can you add such a test here? Just make two identical POs and call uniquepos on them and test that the result has length 1.

Reply:
Yes, it errored if the input was a vector `[]` and not a typed vector `PeriodicOrbit[]`. There is a test for it which was using the typed vector and didn't catch the bug. I modified the test. Is it ok now?